packages/verify-env/src/testSource.txt

```diff
@@ -0,0 +1,91 @@
+import "@rainbow-me/rainbowkit/styles.css";
```

Comment:
I *think* it should be possible to save this as a `ts` but disable typechecking with a directive at the top, if that helps.

Reply:
it is, using `// @ts-nocheck`, but in the interest of not tripping up the rest of my (and others') IDEs, i'm gonna leave it as .txt.